builtin/credential/ldap/path_config_rotate_root_test.go

```diff
@@ -0,0 +1,64 @@
+package ldap
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/vault/helper/testhelpers/ldap"
+	logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+// This test relies on an external ldap server with a suitable person object (cn=admin,dc=planetexpress,dc=com)
+// with bindpassword "admin". - see the backend_test for more details.
```

Comment:
Wasn't able to find backend_test but I might be missing something.

Reply:
These lines: https://github.com/hashicorp/vault/blob/39762174206ee353e8cb2d1eab2c544723b91c2d/builtin/credential/ldap/backend_test.go#L387C1-L387C1

Reply:
Thanks!